code/game/objects/structures/janicart.dm

```diff
@@ -238,14 +238,6 @@
 		step(src, direction)
 		update_mob()
 		handle_rotation()
-		if(istype(src.loc, /turf/simulated))
-			var/turf/simulated/T = src.loc
-			if(T.wet == 2)	//Lube! Fall off!
-				playsound(src, 'sound/misc/slip.ogg', 50, 1, -3)
-				buckled_mob.Stun(7)
-				buckled_mob.Weaken(7)
-				unbuckle_mob()
-				step(src, dir)

```

Comment:
janicart lube slipping is now handled in /handle_slip() like all other slipping.